frameworks/js-bindings/bindings/script/jsb.js

```diff
@@ -111,6 +111,15 @@ require('script/jsb_deprecated.js');
 require('script/jsb_loaders.js');
 require('script/jsb_pool.js');
 
+require('script/studio/jsb_studio_load.js');
+require('script/studio/parsers/jsb_studio_action-1.x.js');
+require('script/studio/parsers/jsb_studio_action-2.x.js');
+require('script/studio/parsers/jsb_studio_scene-1.x.js');
+require('script/studio/parsers/jsb_studio_timelineParser-1.x.js');
+require('script/studio/parsers/jsb_studio_timelineParser-2.x.js');
+require('script/studio/parsers/jsb_studio_uiParser-1.x.js');
+require('script/studio/parsers/jsb_studio_compatible.js');
```

Comment:
This whole section should be required only if user want to use studio parser, so there should be a way to do it. Read project.json is a possible way.

If it takes time, it's acceptable to merge it as it is currently, but please open an issue for later modifications, and all these should be done in 3.3